charts/prometheus-pushgateway/values.yaml

```diff
@@ -213,6 +213,18 @@ containerSecurityContext: {}
 ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
 affinity: {}
 
+## Pod anti-affinity can prevent the scheduler from placing push-gateway replicas on the same node.
+## The default value "soft" means that the scheduler should *prefer* to not schedule two replica pods onto the same node but no guarantee is provided.
```

Comment:
I am not sure about the default value as the present default is an empty string.

```suggestion
## The value "soft" means that the scheduler should *prefer* to not schedule two replica pods onto the same node but no guarantee is provided.
```

Reply:
Ah, sorry. The default value of an empty string will have no affect - no podAntiAffinity configuration will be applied. I'll fix this comment.

Reply:
Done